app/domain/tool_category.rb

```diff
@@ -14,6 +14,11 @@ def news?
     false
   end
 
+  def parent_id
+    return '' if @id.empty?
+    @entity.try(:parent_id)
+  end
+
```

Comment:
I don't understand this method. Why the method exists? And can you add tests to it, please?